content/en/docs/installation/compatibility.md

```diff
@@ -6,7 +6,22 @@ type: "docs"
 ---
 
 Below you will find details on various compatibility issues and quirks that you
-may be effected by in your environment.
+may be affected by when deploying cert-manager. If you believe we've missed something
+please feel free to raise an issue or a pull request with the details!
+
+{{% alert title="Important" color="warning" %}}
+If you run cert-manager using the host's network, be aware that kubelet listens on port `10250`
```

Comment:
Suggestion: perhaps we should also add that by default `cert-manager` is not using host's network? I can see how it's good to make this warning clear, but am a little bit worried that we may be confusing someone who has just opened to page to see if they can run `cert-manager` on GKE etc. Alternatively I guess we could also move the warning to the AWS section or make a separate subsection for running on host's network and link to that from AWS EKS/AWS Fargate sections (probably overcomplicated).

Reply:
yeah I think that could be clearer, good suggestion. I'll take a swing at it.